libs/langchain-community/src/vectorstores/pgvector.ts

```diff
@@ -501,6 +501,15 @@ export class PGVectorStore extends VectorStore {
           parameters.push(..._value.in);
           paramCount += _value.in.length;
         }
+        if (_value.arrayContains) {
+          whereClauses.push(
+            `${
+              this.metadataColumnName
+            }->'${key}' ?| array[${_value.arrayContains
```

Comment:
Ah wait, actually this has some SQL injection risk - can you parameterize?

Reply:
I have addressed this by aligning the code to match how it's done in "IN" filters.